cmd/influx/auth.go

```diff
@@ -21,9 +23,33 @@ func newAuthCommand() cli.Command {
 	}
 }
 
+func helpText(perm string) struct{ readHelp, writeHelp string } {
+	var helpOverrides = map[string]struct{ readHelp, writeHelp string }{
+		"user":      {"perform read actions against organization users", "perform mutative actions against organization users"},
+		"buckets":   {"perform mutative actions against organization buckets", "perform mutative actions against organization buckets"},
+		"telegrafs": {"read telegraf configs", "create telegraf configs"},
+		"orgs":      {"read organizations", "create organizations"},
+		"dbrps":     {"read database retention policy mappings", "create database retention policy mappings"},
+	}
+
+	help := helpOverrides[perm]
+	if help.readHelp == "" {
+		help.readHelp = fmt.Sprintf("read %s", perm)
+	}
+	if help.writeHelp == "" {
+		help.writeHelp = fmt.Sprintf("create or update %s", perm)
+	}
+
+	help.readHelp = "Grants the permission to " + help.readHelp
+	help.writeHelp = "Grants the permission to " + help.writeHelp
+	return help
+}
+
 func newCreateCommand() cli.Command {
 	var params auth.CreateParams
 	flags := append(commonFlags(), getOrgFlags(&params.OrgParams)...)
+
+	// default: create and update foo / reed foo
```

Comment:
```suggestion
	// default: create and update foo / read foo
```
Is this a misplaced comment? Should it go up where `help.readHelp` and `help.writeHelp` are set?